roles/contiv_network/templates/netplugin.j2

```diff
@@ -1 +1,6 @@
-NETPLUGIN_ARGS=""
+{# If an interface is specified for netplugin, use it #}
+{% if netplugin_if is defined %}
+NETPLUGIN_ARGS='-docker-plugin -vlan-if {{netplugin_if}} -vtep-ip {{node_addr}}'
+{% else %}
+NETPLUGIN_ARGS='-docker-plugin -vtep-ip {{node_addr}}'
```

Comment:
not sgtm Divya. We shouldn't put in code that we know will break and the current logic is broken.

Even after netplugin fix, as long as netplugin needs a datapath interface to be passed as argument we need to pass it correctly from ansible.